packages/react-components/react-list-preview/src/components/List/List.test.tsx

```diff
@@ -2,11 +2,18 @@ import * as React from 'react';
 import { render } from '@testing-library/react';
 import { isConformant } from '../../testing/isConformant';
 import { List } from './List';
+import { ListProps } from './List.types';
 
 describe('List', () => {
   isConformant({
-    Component: List,
+    Component: List as React.FunctionComponent<ListProps>,
```

Comment:
Please create an issue for this, it should be debugged as type cast should not be mandatory there.

Reply:
Done, #29943